schemas/fingerprint-server-api.yaml

```diff
@@ -1703,3 +1730,25 @@ components:
       required:
         - code
         - message
+    VelocityIntervals:
+      type: object
+      additionalProperties: false
+      nullable: true
```

Comment:
Under what circumstances would it be null? Can we include it in the object description? 

`VelocityResult` is null if no velocity data at all is generated (because of some error?) 
 
Based on the example it seems  `VelocityIntervalResult` for a `linkedId` can be null if there is no `linkedId` provided. 

But when would `"distinctIp": null` happen?

Reply:
You are correct, basically in our current release `distinctLinkedId` is the only signal which is optional(based on customer traffic) (but going forward there would be many)

All other would have at least a count associated to them

Reply:
I am not sure I understand. This `nullable` here suggests you can get a response like 
```
  "distinctLinkedId": null
```
But is that correct?
Because the example suggested that if you don't have the linked ID data, you would get this instead

```
  "distinctLinkedId": {
      "intervals": null
    },
``` 

So is this `nullable: true` incorrect? And if it's correct, under what circumstances would you get ` "distinctLinkedId": null` ?

Reply:
Got it, thank you!